### ISSUE TITLE
Fails silently if SSID is unusual.

### ISSUE THREAD

**Issue by cire831** [davesteele/comitup#193]
running version 1.2.5

The RPi comes up fine, comitup is active and announcing hotspot comitup-7340.  I connect to it and manually launch a browser to 10.42.0.1

I am presented with one selection for "Familytimevacationrentals.com\x20".   Yes, I know someone configured a space on the end of the SSID name for the access point.   I'll get them to fix this soon as I get the password to the AP.

I select that network and enter the password for the above network.    Nothing happens.   The RPi running comitup is still in HOTSOT mode, announcing comitup-7340.

I have a console on the RPi, so I then used comitup-cli to connect manually

```
dvt3 (5): comitup-cli

State: HOTSPOT
Connection: comitup-7340
Points:
    1: Familytimevacationrentals.com\x20
    2: TP-LINK_9827_almond
Available commands:
    (i)nfo
    (r)eload
    (q)uit
    connect to <n>
    (m)anual connection
command?: 1
password: 
Traceback (most recent call last):
  File "/usr/sbin/comitup-cli", line 11, in <module>
    load_entry_point('comitup==1.2.5', 'console_scripts', 'comitup-cli')()
  File "/usr/share/comitup/cli/comitupcli.py", line 112, in interpreter
    do_connect(ciu_client, points[index-1]['ssid'], password)
  File "/usr/share/comitup/cli/comitupcli.py", line 37, in do_connect
    ciu_client.ciu_connect(ssid, password)
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 145, in __call__
    **keywords)
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.NetworkManager.Settings.Connection.InvalidProperty: 802-11-wireless.ssid: SSID length is out of range <1-32> bytes
dvt3 (6):
```

**Comment by davesteele** [comment]
In debugging, I found that a NetworkManager connection that should create a hotspot with a trailing space actually creates one with a bizarre escape sequence. That's a [wontfix](https://www.goodreads.com/quotes/7570350-the-patient-says-doctor-it-hurts-when-i-do-this).

The escape you are seeing is from the "iw" command, which I use to get the AP list. I fixed this, but, well, it leads to the creation of a NetworkManager connection, which changes it back to the escape sequence in connection name and AP name, which will always fail.

This is a NetworkManager bug. wontfix.
